Unknown CMake command "duatic_add_pytest_test".

Hey I just tried to build the `duatic_dynaarm_single_example` and run into the issue:


```
--- stderr: duatic_common                                                                                                                
CMake Error at CMakeLists.txt:29 (duatic_add_pytest_test):
  Unknown CMake command "duatic_add_pytest_test".

```

All repos where freshly cloned and are on their default branch

Full build output


```
Starting >>> message_logger
Starting >>> soem_vendor
Starting >>> duatic_dynaarm_description
Starting >>> duatic_dynaarm_hardware_interface_base
Starting >>> duatic_dynaarm_msgs
Starting >>> duatic_common
Starting >>> duatic_dynaarm_extensions
Starting >>> duatic_helpers
--- stderr: duatic_common                                                                                                                
CMake Error at CMakeLists.txt:29 (duatic_add_pytest_test):
  Unknown CMake command "duatic_add_pytest_test".


---
Failed   <<< duatic_common [0.65s, exited with code 1]
Aborted  <<< soem_vendor [0.82s]                                                                                                                             
Aborted  <<< duatic_dynaarm_description [0.90s]                                                                                                                                    
Aborted  <<< duatic_dynaarm_extensions [0.92s]
Aborted  <<< duatic_helpers [0.96s]                                                                                                                                         
Aborted  <<< message_logger [1.30s]                                                                                                                                      
Aborted  <<< duatic_dynaarm_hardware_interface_base [1.42s]                                                                           
Aborted  <<< duatic_dynaarm_msgs [2.67s]                            
                                 
Summary: 0 packages finished [2.84s]
  1 package failed: duatic_common
  7 packages aborted: duatic_dynaarm_description duatic_dynaarm_extensions duatic_dynaarm_hardware_interface_base duatic_dynaarm_msgs duatic_helpers message_logger soem_vendor
  3 packages had stderr output: duatic_common message_logger soem_vendor
  8 packages not processed

```

--- a/duatic_common/CMakeLists.txt
+++ b/duatic_common/CMakeLists.txt
@@ -12,6 +12,8 @@ ament_package(
   CONFIG_EXTRAS "duatic_common-extras.cmake"
 )
 
+include(cmake/duatic_package.cmake)
+
 install(
   DIRECTORY cmake
   DESTINATION share/${PROJECT_NAME}

--- a/duatic_common/CMakeLists.txt
+++ b/duatic_common/CMakeLists.txt
@@ -4,7 +4,6 @@ project(duatic_common NONE)
 
 find_package(ament_cmake_core REQUIRED)
 find_package(ament_cmake_python REQUIRED)
-find_package(backward_ros REQUIRED)
 
 ament_python_install_package(duatic_common)
 

--- a/duatic_common/package.xml
+++ b/duatic_common/package.xml
@@ -7,7 +7,6 @@
   <maintainer email="mbloechlinger@duatic.com">Marc Blöchlinger</maintainer>
   <license>BSD 3-Clause</license>
 
-  <depend>backward_ros</depend>
   <depend>launch</depend>
   <depend>launch_ros</depend>
   <depend>osrf_pycommon</depend>

